notebooks/DFT-dataset-generation.ipynb

```diff
@@ -7,31 +7,23 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    loaded = os.path.exists(out_filename) and os.path.getsize(out_filename) == 6985727
</pre><p>HH: Use a proper hash?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore-research/pyscf-ipu/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Good idea - will add to the backlog.